OFL.txt

```diff
@@ -1,8 +1,8 @@
-Copyright (c) 2023 Intel Corp. with Reserved Font Name "Intel One Mono", "IntelOne Mono"
+Copyright 2023 Intel Corp. with Reserved Font Name 'Intel One Mono'
```

Comment:
@FredShallcrass Does Intel have a position on Dave's request to use "Intel" only in the reserved font name as documented in https://github.com/intel/intel-one-mono/pull/27/files#r1248099720?

Also, our policy is to use a different copyright string here: "Copyright 2023 The Intel One Mono Project Authors", then list the authors in the AUTHORS.txt file, as Rosalie proposed in https://github.com/intel/intel-one-mono/pull/27.  Will you please comment on whether this is an acceptable change for your team?  

cc @rosawagner @davelab6